rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -256,4 +256,21 @@ public static void addInputFromMatchingOutputScript(BtcTransaction transaction,
         searchForOutput(outputs, expectedOutputScript)
             .ifPresent(transaction::addInput);
     }
+
+    public static TransactionWitness addBaseWitnessToBtcTxInputThatSpendsFromRedeemScript(Script redeemScript) {
+        int pushForOpNotif = 1;
+        int pushForRedeemScript = 1;
+        int witnessSize = pushForOpNotif + pushForRedeemScript + redeemScript.getNumberOfSignaturesRequiredToSpend();
+
+        TransactionWitness txWitness = new TransactionWitness(witnessSize);
+
+        for (int i = 0; i < witnessSize - 1; i++) {
+            txWitness.setPush(i, new byte[72]);
+        }
+
+        int redeemScriptIndex = witnessSize - 1;
+        txWitness.setPush(redeemScriptIndex, redeemScript.getProgram());
+        return txWitness;
+
+    }
```

Comment:
```suggestion
    public static TransactionWitness createBaseWitnessThatSpendsFromRedeemScript(Script redeemScript) {
        int pushForOpNotif = 1;
        int pushForRedeemScript = 1;
        int witnessSize = pushForOpNotif + pushForRedeemScript + redeemScript.getNumberOfSignaturesRequiredToSpend();

        TransactionWitness txWitness = new TransactionWitness(witnessSize);

        for (int i = 0; i < witnessSize - 1; i++) {
            txWitness.setPush(i, new byte[72]);
        }

        int redeemScriptIndex = witnessSize - 1;
        txWitness.setPush(redeemScriptIndex, redeemScript.getProgram());
        
        return txWitness;
    }
```

you are not actually _adding_ the witness to the input, right?